packages/lwt_ppx/lwt_ppx.2.0.0/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+
+synopsis: "PPX syntax for Lwt, providing something similar to async/await from JavaScript"
+
+version: "2.0.0"
+license: "MIT"
+homepage: "https://github.com/ocsigen/lwt"
+doc: "https://ocsigen.org/lwt/api/Ppx_lwt"
+bug-reports: "https://github.com/ocsigen/lwt/issues"
+
+authors: [
+  "Gabriel Radanne"
+]
+maintainer: [
+  "Anton Bachin <antonbachin@yahoo.com>"
+]
+dev-repo: "git+https://github.com/ocsigen/lwt.git"
+
+depends: [
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.1"}
```
According to its dune-project file, this package requires at least dune 1.1

Reply:
Indeed, in fact this package's `dune` file works with Dune 1.0, but it shares `dune-project` with package `lwt`.